pkg/release/release.go

```diff
@@ -34,6 +34,32 @@ const (
 	bazelVersionPath  = "bazel-genfiles/version"
 	dockerVersionPath = "version"
 	tarballExtension  = ".tar.gz"
+
+	// GCSStagePath is the directory where release artifacts are staged before
+	// push to GCS.
+	GCSStagePath = "_output/gcs-stage"
```

Comment:
Can we parameterize `_output` and then compose the below paths based on that variable?